sys/include/net/gnrc/netif.h

```diff
@@ -75,6 +78,22 @@ typedef struct {
      * @see net_gnrc_netif_flags
      */
     uint32_t flags;
+#if defined(MODULE_GNRC_NETIF_EVENTS) || DOXYGEN
+    /**
+     * @brief   Event queue for asynchronous events
+     */
+    event_queue_t evq;
+    /**
+     * @brief   Pointer to ISR event
+     *
+     * This pointer gives a way to allocate ISR events on the stack of each
+     * gnrc_netif thread, instead of having a single global instance which will
+     * not work if the system has more than one network interface.
+     * The _event_cb function of gnrc_netif.c uses this pointer to be able to
+     * post events from interrupt context.
```

Comment:
Different topic: why don't you just expose the type and allocate it here in full instead of a pointer?

Reply:
Ok, the current comment makes it clearer.